theano/compile/function_module.py

```diff
@@ -496,6 +500,73 @@ def __copy__(self):
                 there.data = here.data
         return cpy
 
+    def copy(self, share_memory=False):
+        """
+        Copy this function. Copied function will have separated maker and
```

Comment:
The second sentence isn't always True. Can you reread the full docstring and make sure it is up to date and exact?